client/jetpack-cloud/sections/agency-dashboard/sites-overview/style.scss

```diff
@@ -46,10 +46,30 @@
 }
 .sites-overview__page-title-container {
 	display: none;
+
 	@include breakpoint-deprecated( ">660px" ) {
 		display: flex;
 		align-items: center;
 	}
+
+	&.is-sticky {
+		position: fixed;
+		width: calc(100% - var(--sidebar-width-max));
+		left: var(--sidebar-width-max);
+		top: var(--masterbar-height);
+		background-color: rgba(246, 247, 247, 0.95);
+		box-shadow: 0 4px 4px 0 rgb(0 0 0 / 8%);
```

Comment:
Let's change this to `box-shadow: 2px 2px 2px 0 rgb(0 0 0 / 8%);` so the shadow doesn't "bleed" into the sidebar, please!